src/component-store/tests/provide-component-store.test.ts

```diff
@@ -0,0 +1,22 @@
+import '@angular/compiler';
+import { test } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { ImmerComponentStore, provideImmerComponentStore } from 'ngrx-immer/component-store';
+import { provideComponentStore } from '@ngrx/component-store';
+
+
+test('provideImmerComponentStore() should equal provideComponentStore()', () => {
+	const ngrxProviders = provideComponentStore(DummyImmerComponentStore as any);
+
+	const ngrxImmerProviders = provideImmerComponentStore(DummyImmerComponentStore);
+
+	// assert
+	assert.instance(ngrxImmerProviders, Array);
+	assert.is.not(ngrxImmerProviders.length, 0);
+	assert.equal(ngrxImmerProviders.toString(), ngrxProviders.toString());
```

Comment:
Because the items in the providers arrays are difference instances I was adding code to check each one it but felt like the test was starting to be too coupled to what ngrx was outputting. So I settled on just toString() and making sure they were the same, its not ideal but I think it does the job.

Reply:
This is fine for me, thanks 👍

Reply:
👍 all updated now.